### ISSUE TITLE
legyen külön kategória az egyházi és a nagy fesztiválokra, és tematikus napokra is

### ISSUE THREAD

**Issue by Kundalata** [rrd108/krisnavolgy#118]
hogy pl a csoport zárolt napoknál külön lehessen megjelenítve

**Comment by rrd108** [comment]
Ezt elég lenne márciusban?

**Comment by Kundalata** [comment]
Igen, elég lesz. De esetleg addig a fesztivál felsorolást (mikor nem lehet csoport programot rendelni) le lehetne a legaljára tenni, vagy el sí tüntetni. Elrettentő nekem ez. Sok sok dátum,...
